### PR TITLE
Maternal disorders YLDs

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

**Review by plinds — APPROVED** [review]

File: docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst [hunk 1]
```diff
@@ -229,6 +233,11 @@ Including,
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * :ref:`Pregnancy model <other_models_pregnancy>`
+
+.. todo::
+
+  Clarify with software engineers if updates were made to the pregnancy model to accomodate the accrual of maternal disorders YLDs (a timestep long post-maternal disorders state in which anemia YLDs are not accrued?). If so, then add these updates to the pregnancy model documentation.
```
Comment (mattkappel) [inline docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst:239]: We did do this. We have two "prepostpartum" states `maternal_disorder` and `no_maternal_disorder`, former being simulants with non-fatal maternal disorders and YLDs and the latter is "normal" no disorder.

File: docs/source/gbd2019_models/causes/maternal_disorders/index.rst [hunk 2]
```diff
@@ -288,21 +288,20 @@ Years of life lost (YLLs) should be assigned to simulants who experience a death
...
 
 .. math::
 
-  DW_\text{c366} = \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - (ACMR - csmr_\text{c366} + csmr_\text{c366} / incidence_\text{c366})}
+  YLDs per non-fatal maternal disorders case = \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - (ACMR - csmr_\text{c366} + csmr_\text{c366} / incidence_\text{c366})}
```
Comment (mattkappel) [inline docs/source/gbd2019_models/causes/maternal_disorders/index.rst:295]: Are `{incidence_rate}_{c366}` and `incidence_\text{c366}` the same thing?

**Review by mattkappel — APPROVED** [review]

File: docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst (hunk 1 above)
Comment (alibow) [reply docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst:239]: Awesome, thank you for clarifying. I will add this to the docs!

File: docs/source/gbd2019_models/causes/maternal_disorders/index.rst (hunk 2 above)
Comment (alibow) [reply docs/source/gbd2019_models/causes/maternal_disorders/index.rst:295]: Yes!

**Pull request merged**